### PR TITLE
Add quotation marks to timing entry names.

### PR DESCRIPTION
**Pull request by quantheory**
This is a (currently untested) variation on #415.

### CODE REVIEW

**Comment by jayeshkrishna** [comment]
As suggested in the PR comments it looks like this change was not tested.
I tried the following case on blues with manually changing SAVE_TIMING="TRUE" (default "FALSE"),
```
./create_newcase -case test_pr467 -res f45_g37_rx1 -compset A
```
The case.run fails with the following error message,
```
timing dir is /lcrc/project/ACME
Traceback (most recent call last):
  File "/var/spool/torque/mom_priv/jobs/1392334.bmgt1.lcrc.anl.gov.SC", line 114, in <module>
    _main_func(__doc__)
  File "/var/spool/torque/mom_priv/jobs/1392334.bmgt1.lcrc.anl.gov.SC", line 109, in _main_func
    success = case_run(case)
  File "/blues/gpfs/home/jayesh/acme/ESMCI_cime/utils/python/CIME/case_run.py", line 409, in case_run
    save_timing_setup_acme(case, lid)
  File "/blues/gpfs/home/jayesh/acme/ESMCI_cime/utils/python/CIME/case_run.py", line 164, in save_timing_setup_acme
    full_timing_dir = os.path.join(timing_dir, "performance_archive", getpass.getuser(), base_case, lid)
  File "/soft/python/2.7.6/lib/python2.7/posixpath.py", line 75, in join
    if b.startswith('/'):
AttributeError: 'NoneType' object has no attribute 'startswith'

```

**Comment by jayeshkrishna** [comment]
Debugging the cause of the issue (made sure that I have "/lcrc/projects/ACME/performance_archive/jayesh" directory. But no case directories were created inside this dir)

**Comment by jedwards4b** [comment]
Is SAVE_TIMING_DIR defined in config_machines.xml?

**Comment by jayeshkrishna** [comment]
Yes (SAVE_TIMING_DIR = /lcrc/project/$PROJECT in env_run.xml), and the comments in the output shows that timing dir (/lcrc/project/ACME) is set.

**Comment by jayeshkrishna** [comment]
Where should CASEBASEID be defined?

**Comment by jgfouca** [comment]
By create_newcase I'd think

**Comment by jgfouca** [comment]
You might want to rebase your branch onto latest master.

File: scripts/Tools/getTiming [hunk 1]
```diff
@@ -86,7 +86,7 @@ class TimingParser:
         nprocs = 0
         ncount = 0
 
-        heading = heading_padded.strip()
+        heading = '"' + heading_padded.strip() + '"'
```
Comment (jayeshkrishna) [inline scripts/Tools/getTiming]: The TimingParser class has been moved from scripts/Tools/getTiming to CIME.get_timing . So this change needs to be manually merged

Comment (jayeshkrishna) [inline scripts/Tools/getTiming]: I have already done manual merge of this change and it looks good

**Comment by jayeshkrishna** [comment]
Tried manual merge of this branch, the run was successful and the new timing stats have quotes around routine names (as expected),
```
d  )   wallmin (proc   thrd  )
"CPL:INIT"                    8        8 1.600000e+01   7.438651e+01     9.299 (     1      0)     9.298 (     5      0)
"CPL:RUN_LOOP_BSTART"         8        8 8.000000e+00   6.136894e-04     0.000 (     0      0)     0.000 (     3      0)
"CPL:RUN_LOOP"                8        8 1.920000e+03   2.417228e+01     3.030 (     1      0)     3.013 (     2      0)
"CPL:CLOCK_ADVANCE"           8        8 1.920000e+03   1.734562e-01     0.023 (     2      0)     0.021 (     4      0)
...
```

**Comment by jayeshkrishna** [comment]
Running CIME regression tests (scripts_regression_tests.py) on blues now

**Comment by jayeshkrishna** [comment]
This dropped off my radar. I will run the regression test suite again and issue a PR

**Comment by jayeshkrishna** [comment]
There is a new branch jayeshkrishna/conflict-resolve-br/santos/quote-timing that resolves the conflicts in this branch and can be merged without conflicts to master

**Comment by jayeshkrishna** [comment]
Continued in PR #906

**Pull request merged**